### PR TITLE
Stakely Pledge

### PR DESCRIPTION
**Pull request by iicc1**

### CODE REVIEW

File: Stakely/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Agoric Validator Pledge
+ 
+
```
Comment (jeetraut) [inline Stakely/README.md:3]:
```suggestion
[Stakely Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper1mxhgvj2c93xahahx9d9fc7rwwtufqza5cn6uhn)
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

**Comment by iicc1** [comment]
We created a page with information about Agoric and a staking tutorial in both English and Spanish. It contains a brief introduction to Agoric, with links and a simple staking calculator tool that will work when the staking rewards start. It also contains a step-by-step staking tutorial and frequently asked questions regarding Agoric staking.

- English: https://stakely.io/en/agoric-staking
- Spanish: https://stakely.io/es/agoric-staking

More coming soon!

**Comment by iicc1** [comment]
We added Agoric to our load balancer for web3 APIs. This free service aggregates several public and private nodes and the requests are routed among them. It has many features like health checks, retires on errors, CORS-enabled requests, caching, etc. It can be used for production-ready apps. We also deployed our own Agoric RPC and LCD and connected them to the tool

https://agoric-rpc.stakely.io --> Aggregates 10 Agoric public nodes
https://agoric-lcd.stakely.io --> Aggregates 6 Agoric public nodes

**Pull request merged**